CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to Singleton Java client
+
+
+## Welcome
+
+Welcome to Singleton Java client! This guide provides information on filing issues and guidelines for open source contributors. **Please leave comments / suggestions if you find something is missing or incorrect.**
+
+## Get Started

```

Comment:
Looks like there are some overlaps with this doc: https://github.com/vmware/singleton/blob/master/CONTRIBUTING.md
so, I would suggest separate the common part and give above link for it, and this guideline only includes Java client specific.

Reply:
The doc has already been refered to.